src/Components/Samples/Accessibility/Accessibility.csproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
```

Comment:
The samples are meant to be sandboxes for us to try things locally. https://github.com/aspnet/samples.